README.md

```diff
@@ -1 +1,56 @@
-# AlumLink
\ No newline at end of file
+# AlumLink
+Development Setup
```

Comment:
Make sure you're using proper headings in the right places in the documentation. You want this to be easy for someone to read and jump through.